Deeploy/Targets/NE16/__init__.py

```
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2024 ETH Zurich and University of Bologna
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from . import *
```

Ruff (0.15.9)

[error] `from . import *` used; unable to detect undefined names

(F403)